x-pack/plugins/ml/server/routes/job_service.ts

```diff
@@ -294,8 +294,8 @@ export function jobServiceRoutes({ router, routeGuard }: RouteInitialization) {
     routeGuard.fullLicenseAPIGuard(async ({ client, mlClient, request, response }) => {
       try {
         const { createFullJobsList } = jobServiceProvider(client, mlClient);
-        const { jobIds } = request.body;
-        const resp = await createFullJobsList(jobIds);
+        const { jobIds, excludeGenerated } = request.body;
```

Comment:
As above, did you mean to include these changes? Looks like they are part of the `exclude_generated` work.

Reply:
This is now reverted https://github.com/elastic/kibana/pull/88880/commits/311892828ba8ad9bf92efe643c97c43d7c0439a1